lib/utils/promote.ts

```diff
@@ -29,14 +29,6 @@ export async function join(
 	deviceHostnameOrIp?: string,
 	appName?: string,
 ): Promise<void> {
-	logger.logDebug('Checking login...');
-	const isLoggedIn = await sdk.auth.isLoggedIn();
-	if (!isLoggedIn) {
-		logger.logInfo("Looks like you're not logged in yet!");
-		logger.logInfo("Let's go through a quick wizard to get you started.\n");
-		await runCommand('login');
```

Comment:
Running 'login' in this way was not working for some reason: on macOS, it would print the CLI 'help' and then run login, and on Windows, it was failing altogether with "BalenaRequestError: Request error: Invalid binding". I spent a bit of time trying to figure it out but gave up (not high enough of a priority) and decided it was better to just add `permission: 'user'` in `join.ts`, which is how every other Capitano command is currently implemented in relation to the requirement of logging in, including the `balena leave` command.